src/_defaults.scss

```diff
@@ -130,15 +130,16 @@ h1, .h1, h2, .h2, h3, .h3, h4, .h4, h5, .h5, h6, .h6
 
 /* Exclude temporary the form element from the font update, keep same style as before
  * - Rational: More testing and various adjustment need to be completed before to apply it
+ * - In the mean time, users can apply the class "force-style-gcweb-current" to the form
  */
-:not(#wb-srch) form, .checkbox, .checkbox-inline, .radio, .radio-inline, form .btn:not(.btn-call-to-action) {
+:not(#wb-srch) form:not(.force-style-gcweb-latest), :not(.force-style-gcweb-latest) .checkbox, :not(.force-style-gcweb-latest) .checkbox-inline, :not(.force-style-gcweb-latest) .radio, :not(.force-style-gcweb-latest) .radio-inline, form:not(.force-style-gcweb-latest) .btn:not(.btn-call-to-action) {
 	font-size: 16px;
 	line-height: 23px;
 }
-form .btn:not(.btn-call-to-action), .form-control {
+form:not(.force-style-gcweb-latest) .btn:not(.btn-call-to-action), :not(.force-style-gcweb-latest) .form-control {
 	padding: 6px 12px;
 }
-legend {
+:not(.force-style-gcweb-latest) legend {
 	line-height: 1.65em;
 }
```

Comment:
@duboisp Should this selector be dropped?

GCWeb already contains a selector that sets the ``main`` element's default ``line-height`` to 1.65em:
https://github.com/wet-boew/GCWeb/blob/7b2164a252d4313b00be3d4964668fcf6e0c06f5/src/_defaults.scss#L22

So assuming GCWeb doesn't use any legends outside of its ``main`` element, this selector (both the old and updated versions) is redundant.